src/App.js

```diff
@@ -1,19 +1,56 @@
 import React from 'react';
 import './App.css';
-
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoForm } from './components/TodoForm/TodoForm';
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId),
+}));
+
+class App extends React.Component {
+  state = {
+    todos: preparedTodos,
+  }
+
+  addNewTodo = (title, name) => {
+    this.setState((prevState) => {
+      const newTodo = {
+        userId: name.id,
```

Comment:
name is a string, how could it have a prop id?

Reply:
userId is empty

![image](https://user-images.githubusercontent.com/57598825/105821784-e7f84100-5fc3-11eb-95bd-43525c974ee7.png)

Reply:
mb save userId to the Todo form state or the whole user and then pass it's userId to the addTodo